onebusaway-android/src/main/java/org/onebusaway/android/directions/util/LocationUtil.java

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
Let's get rid of this class and move the methods (minus `getRegionBounds(ObaRegion region)`) to the existing `LocationUtils` class.  I think they are generally useful and there isn't a good reason to split them into a separate utility class.